.github/workflows/tests-merge.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
         with:
-          version: nightly
+          version: 0.2.0
```

Comment:
I think this will be a long long time ago. Generally for foundry pin to a hash from here https://github.com/foundry-rs/foundry/releases